backend/internal/controller/repository/post_repository.go

```diff
@@ -142,6 +142,24 @@ func (r *PostRepository) Get(id int) (*entity.Post, error) {
 	return convertPostRepositoryModelToEntity(&post, &user), nil
 }
 
+func (r *PostRepository) DeletePost(id int) error {
+	var post model.Post
+	postResult := r.Conn.Where("id = ?", id).First(&post)
+	if postResult.Error != nil {
+		if errors.Is(postResult.Error, gorm.ErrRecordNotFound) {
+			return nil
+		}
+		return postResult.Error
+	}
+
+	postUpdateResult := r.Conn.Delete(&post)
```

Comment:
[nits]
ここもしかしたら私の実装の残骸かもなんだけど、`postDeleteResult`とかの方が正確かも！
```suggestion
	postDeleteResult := r.Conn.Delete(&post)
```

Reply:
間違えた、こっちでやっちゃった
[fix: typo](https://github.com/givery-bootcamp/dena-2024-team1/pull/109/commits/a4fb7906b8d7b0ba7d14264135871f90ba60afb6)